projects/kit/components/slider/slider.style.less

```diff
@@ -157,8 +142,7 @@
     background-clip: content-box; /* To clip filling of paddings */
     outline: none;
     border-radius: var(--tui-radius-m);
-    // TODO: Uncomment it in v4.0
-    // margin: 0; /* Drop browser default margins */
+    margin: 0; /* Drop browser default margins */
```

Comment:
Delete this lines too in this PR (to see that all screenshots are still fine).

https://github.com/taiga-family/taiga-ui/blob/34e421528407c395e6cbad4fa2bb24902075f898/projects/kit/components/range/range.style.less#L131-L132

I've forgotten about it during the first review, sorry!

Reply:
fixed